app/experimenter/docs/adrs/0006-doc-hub-repo.md

```diff
@@ -0,0 +1,98 @@
+
+This decision would have us use the `mozilla/experimenter` repository, but restrict all code and documentation to a dedicated branch. The dedicated branch would first entirely remove all code coming from the `main` branch, never merging back into it. Built/static code would then either be committed to and served from that same branch or a different branch (e.g. `docs-hub` for development, `gh-pages` for static).
+
+Pros
+
+- Where using the main branch meant configuring CircleCI to not run certain workflows if changes only occurred in documentation files, a dedicated branch means we could entirely remove the CircleCI config. Assuming CircleCI evaluates the config file on each push, if one does not exist it could mean that it won't run at all, mitigating the issue altogether.
+
+Neutral
+
+- This option inherits pros and cons from the two other options, as it is simultaneously part of and not part of `mozilla/experimenter`. It gives us that "fresh" repository setup, while allowing us to use some of the existing integrations. However it also imposes the same idea that it is scoped to the Experimenter Console, adds cultural overhead, has potential CI implications, and introduces its own unique cons.
+
+Cons
+
+- It's effectively a new repository, hidden inside an existing repository, and likely has the greatest learning curve for less-technical contributors.
+- Having active development/code and automated commits/static files in the same branch would be messy, but having _two_ dedicated branches for documentation (one for active development, one for static files) would be cumbersome and likely confusing.
+- You would not be able to merge changes to both documentation and the Experimenter codebase in the same Pull Request.
+- Would mean mixed PRs -- Experimenter and Docs Hub -- that are talking about completely distinct source paths.
+- Precludes us from using the repository's GitHub Pages path for another project that might be more specific to Experimenter Console.
+
+### Set up a new, dedicated repository
+
+This decision would have us create a new repository under the Mozilla GitHub org, separate from any other codebase. The active code and documentation files would live in the `main` branch based at the root, and the built/static files would be committed to and served from a `gh-pages` branch.
+
+Pros
+
+- It's a "fresh" repository, allowing us greater initial flexibility in how code is set up, organized, and deployed. Without needing to be shuffled around to accommodate larger changes to the Experimenter Console code it also means greater long-term flexibility.
+- Deploys could be done any way we see fit, and would not need to be shoehorned into a CircleCI job (not that CircleCI would be a bad fit for this). For example, a simple GitHub Actions workflow could be used.
+- Provides us with a neutral spot for documentation across all projects associated with Experimenter (Nimbus UI, SDKs, Jetstream, etc).
```

Comment:
Is there a pro about CI runs faster / easier for non code folks check it out / edit  v.s. all of experimenter?